mama/c_cpp/src/c/subscription.c

```diff
@@ -1550,31 +1560,31 @@ mama_status mamaSubscription_deactivate_internal(mamaSubscriptionImpl *impl)
 
 void MAMACALLTYPE mamaSubscription_DestroyThroughQueueCB(mamaQueue Queue, void* closure)
```

Comment:
All of these WITH_ENTITLEMENTS changes provide Window compile compat. Same question as mempool.c